mod-calendar/src/main/resources/bama/mod-calendar/features/calendarPeriods.feature

```diff
@@ -213,26 +222,15 @@ Feature: Calendar periods
     When method POST
     Then status 201
 
-  Scenario: POST calendar period by service point id should return 422 if period is invalid
```

Comment:
Error messages for improper payloads now vary (and do not return the WebDAV specific code 422).  All that matters is that it properly indicates an error (>= 400), which it does.